tests/unit/test_power_grid_model.py

```diff
@@ -13,10 +13,11 @@
     DatasetType,
     PowerGridModel,
     initialize_array,
+    power_grid_meta_data,
 )
 from power_grid_model._utils import compatibility_convert_row_columnar_dataset
 from power_grid_model.errors import InvalidCalculationMethod, IterationDiverge, PowerGridBatchError, PowerGridError
-from power_grid_model.utils import get_dataset_scenario
+from power_grid_model.utils import get_dataset_scenario, json_serialize_to_file
```

Comment:
do you still need this?
```suggestion
from power_grid_model.utils import get_dataset_scenario
```

Reply:
Addressed.